src/mem/cache/tags/tagged_entry.hh

```diff
@@ -114,44 +102,34 @@ class TaggedEntry : public ReplaceableEntry
             isSecure(), isValid(), ReplaceableEntry::print());
     }
 
-  protected:
-    /**
-     * Set tag associated to this block.
-     *
-     * @param tag The tag value.
-     */
-    virtual void setTag(Addr tag) { _tag = tag; }
+    bool
```

Comment:
Is this reachable given the existence of line 131?

Reply:
probably not.. but I will try out removing it in another PR.